sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/ServiceSpec.java

```diff
@@ -31,5 +32,5 @@
     String getZookeeperConnection();
 
     @JsonProperty("replacement-failure-policy")
-    ReplacementFailurePolicy getReplacementFailurePolicy();
+    Optional<ReplacementFailurePolicy> getReplacementFailurePolicy();
```

Comment:
To clarify from earlier comment -- does this represent a compatibility break in ServiceSpec serialization? I don't remember for sure but an optional may be serialized differently from a nullable.